tests/test-extensions/test-oauth-extension/src/main/java/org/mule/test/oauth/TestOAuthOperations.java

```diff
@@ -55,6 +56,21 @@ public void tokenExpired(@Connection TestOAuthConnection connection) {
     }
   }
 
+  @MediaType(TEXT_PLAIN)
+  public void tokenExpiredAsync(@Connection TestOAuthConnection connection,
```

Comment:
and the assertion?

Reply:
https://github.com/mulesoft/mule-integration-tests/pull/1696 the tests are in this PR in the mule-integration-tests repo